test/cmd/images_tests.sh

```diff
@@ -21,7 +21,8 @@ os::test::junit::declare_suite_start "cmd/images${IMAGES_TESTS_POSTFIX:-}"
 # some steps below require that we use system:admin privileges, but we don't
 # want to stomp on whatever context we were given when we started
 original_context="$( oc config current-context )"
-os::cmd::expect_success 'oc login -u system:admin'
+os::cmd::try_until_text "oc get project" "${project}"
```

Comment:
shouldn't be needed... the things launching this script are already waiting until we have access